pkg/apis/core/validation/validation.go

```diff
@@ -3185,6 +3191,8 @@ type PodValidationOptions struct {
 	AllowMultipleHugePageResources bool
 	// Allow pod spec to use hugepages in downward API
 	AllowDownwardAPIHugePages bool
+	// Validates pod-deletion-cost annotation value.
+	ValidatePodDeletionCost bool
```

Comment:
flip this so the zero-value validates and requires explicit opting into allowing invalid data (`AllowInvalidPodDeletionCost`)

Reply:
you mean if for some reason this is not initialized via `GetValidationOptionsFromPodSpecAndMeta`?

Reply:
done.